ui/components/institutional/note-to-trader/index.scss

```diff
@@ -0,0 +1,33 @@
+.note-header {
+  display: flex;
+  justify-content: space-between;
```

Comment:
Since, note-header is using Box, I think you can remove it and use the  following utility props directly in component               
```
display={DISPLAY.FLEX}
justifyContent={JustifyContent.spaceBetween}
```